package.json

```diff
@@ -14,10 +14,17 @@
     "cluster:clean": "d2 cluster down 2.38 --clean",
     "test": "d2-app-scripts test",
     "deploy": "d2-app-scripts deploy",
-    "format": "prettier . -w"
+    "format": "prettier --write ./src",
+    "lint": "eslint --fix ./src",
+    "prepare": "husky install"
```

Comment:
Do you think it makes sense to add a github action that will do a lint test on any PR and fail if the linting fails?

Reply:
Will add github action in a separate PR.